### PR TITLE
OPEN-486: Add new file formats (for FGP)

### PR DESCRIPTION
**Pull request by RabiaSajjad**

### CODE REVIEW

File: ckanext/canada/schemas/presets.yaml [hunk 1]
```diff
@@ -3230,6 +3230,9 @@ presets:
       mimetype: application/x-filegdb
     - value: Flat raster binary
       replaces: [flat raster binary]
+    - value: GDB
+      mimetype: application/x-filegdb
+      label: GeoSoftDatabases
```
Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]: We already have an entry for "FGDB/GDB"

File: ckanext/canada/schemas/presets.yaml [hunk 2]
```diff
@@ -3252,6 +3255,9 @@ presets:
       mimetype: application/gml+xml
       openness_score: 3
       replaces: [gml]
+    - value: GRD
+      mimetype: application/x-filegdb
```
Comment (wardi) [inline ckanext/canada/schemas/presets.yaml]: is this the right mimetype?

File: ckanext/canada/schemas/presets.yaml (hunk 1 above)
Comment (wardi) [reply ckanext/canada/schemas/presets.yaml]: @PatLittle we have a bunch of these geospatial formats now that have unilingual labels that contain product names. Should we go back to using "file extension (+ description as required)" or is it ok to have unilingual labels for this controlled list?

File: ckanext/canada/schemas/presets.yaml (hunk 2 above)
Comment (RabiaSajjad) [reply ckanext/canada/schemas/presets.yaml]: fixed

**Pull request merged**